### PR TITLE
MAUI-[Others]-UG documentation for localization.

### PR DESCRIPTION
**Pull request by sudarsanMS**
### Description

- UG documentation for localization.

### CODE REVIEW

File: maui-toc.html [hunk 1]
```diff
@@ -523,6 +523,7 @@
 			<li><a href="/maui/DataGrid/unbound-column">Unbound Column</a></li>
 			<li><a href="/maui/DataGrid/export-to-excel">Export to Excel</a></li>
             <li><a href="/maui/DataGrid/export-to-pdf">Export to PDF</a></li>
+            <li><a href="/maui/DataGrid/localization">Export to PDF</a></li>
```
Comment (ashok-kuvaraja) [inline maui-toc.html:526]: The heading title should be **Localization**. Modify the changes.

File: MAUI/DataGrid/localization.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+
+		SfDataGridResources.ResourceManager = new ResourceManager("ResxPath", Application.Current.GetType().Assembly);
+	   MainPage = new MainPage();
+	}
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Unbound Rows" src="Images\localization\maui-datagrid-localization.png" width="404"/>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/localization.md:45]: Alt text should be `Load more in MAUI SfDataGrid`

File: MAUI/DataGrid/localization.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+   1. Create new folder, named as `Resources` in the application.
+
+   2. Right-click on the `Resources` folder, select `Add` and then `NewItem.`
+
+   3. In Add New Item wizard, select the Resource File option and name the filename as `SfDataGrid.<culture name>.resx.` For example, give the name as `SfDataGrid.fr-FR.resx` for French culture.
+
+   4. The culture name indicates the name of the language and country.
+
+   <img alt="Unbound Rows" src="Images\localization\maui-datagrid-localization-create-resource-file.png" width="404"/>
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/localization.md:62]: Modify the alt texts in all the places.

File: MAUI/DataGrid/localization.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+
+namespace MauiDataGridDemo;
+public partial class App : Application
+{
+	public App()
+	{
+		InitializeComponent();
+		CultureInfo.CurrentUICulture = new CultureInfo("fr-FR");
+      //// ResXPath => Full path of the resx file; For example : //SfDataGridResources.ResourceManager = new ResourceManager
+      // ("MauiDataGridDemo.Resources.SfDataGrid", Application.Current.GetType().Assembly);
```
Comment (ashok-kuvaraja) [inline MAUI/DataGrid/localization.md:35]: Remove the above commented lines and change the highlight line value.

**Review by ashok-kuvaraja — CHANGES_REQUESTED** [review]

File: MAUI/DataGrid/localization.md (hunk 4 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/localization.md:35]: modified

File: MAUI/DataGrid/localization.md (hunk 3 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/localization.md:62]: modified

File: MAUI/DataGrid/localization.md (hunk 2 above)
Comment (sudarsanMS) [reply MAUI/DataGrid/localization.md:45]: modified

File: maui-toc.html (hunk 1 above)
Comment (sudarsanMS) [reply maui-toc.html:526]: modified

**Review by ashok-kuvaraja — APPROVED** [review]
The changes are fine.

**Pull request merged**